jenkins/spark-tests.sh

```diff
@@ -99,6 +99,12 @@ tar xzf "$RAPIDS_INT_TESTS_TGZ" -C $ARTF_ROOT && rm -f "$RAPIDS_INT_TESTS_TGZ"
 $MVN_GET_CMD -DremoteRepositories=$SPARK_REPO \
     -DgroupId=org.apache -DartifactId=spark -Dversion=$SPARK_VER -Dclassifier=bin-hadoop3.2 -Dpackaging=tgz
 
+# Donwload parquet-hadoop jar for parquet-read encryption tests
+PARQUET_HADOOP_VER=`mvn help:evaluate -q -N -Dexpression=parquet.hadoop.version -DforceStdout -Dbuildver=${SHUFFLE_SPARK_SHIM/spark/}`
+[[ "$(printf '%s\n' "1.12.0" "$PARQUET_HADOOP_VER" | sort -V | head -n1)" = "1.12.0" ]] && \
```

Comment:
This needs to check for a version > 1.12 rather than exactly equal to 1.12.  Spark 3.2.1 uses 1.12.2, for example. 
 See the effort to do this in #5994.

Reply:
Yes, this script `[[ "$(printf '%s\n' "1.12.0" "$PARQUET_HADOOP_VER" | sort -V | head -n1)" = "1.12.0" ]]` is to make sure `version > 1.12`, e.g.  



PARQUET_HADOOP_VER=`1.10.0`, then `sort -V` looks like:
```
1.10.0 --> (head -n1)
1.12.0

``` --> NOT Download

PARQUET_HADOOP_VER=`1.12.0`, then `sort -V` looks like:
```
1.12.0 --> (head -n1)
1.12.0

``` --> Download

PARQUET_HADOOP_VER=`1.12.1`, then `sort -V` looks like:
```
1.12.0 --> (head -n1)
1.12.1

``` --> Download

Reply:
The logic is the same as https://github.com/NVIDIA/spark-rapids/pull/5994.

It checks the parquet_hadoop jar file name (with version in it)

Here checks there parquet_hadoop version

Reply:
Ah sorry, misread the code. 🤦